docs/guide/concept-di-container.md

```diff
@@ -224,11 +224,11 @@ and the container will automatically resolve dependencies by instantiating them
 them into the newly created objects. The dependency resolution is recursive, meaning that
 if a dependency has other dependencies, those dependencies will also be resolved automatically.
 
-You can use [[yii\di\Container::get()]] to create new objects. The method takes a dependency name,
+You can use [[yii\di\Container::get()|get()]] to create new objects. The method takes a dependency name,
```

Comment:
I think "to create new objects" is not accurate and a bit confusing since there's `setSingleton()`. I'd say "to either create or get object instance".